jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml

```diff
@@ -88,6 +88,7 @@
                 export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
                 export GINKGO_PARALLEL="y"
                 export PROJECT="e2e-gce-gci-ci-master"
+                export KUBE_MASTER_OS_DISTRIBUTION="gci"
```

Comment:
I believe `KUBE_NODES_OS_DISTRIBUTION` defaults to `KUBE_MASTER_OS_DISTRIBUTION` so we don't have to specify it here?